src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -479,12 +485,32 @@ redeemADA :: WalletWebMode ssc m => SendActions m -> CWalletRedeem -> m CTx
 redeemADA sendActions CWalletRedeem {..} = do
     let base64rify = T.replace "-" "+" . T.replace "_" "/"
```

Comment:
Add a comment that it's a safety measure just in case we used base64url somewhere before please. :)